R/reportRenovation.R

```diff
@@ -25,130 +37,151 @@ reportRenovation <- function(gdx, brickSets = NULL, silent = TRUE) {
 
 
 
+  # FUNCTION -------------------------------------------------------------------
+
+  # Replace the pattern if it is a separate vector entry
+  .replaceInVec <- function(pattern, replacement, char) {
+    sub(paste0("^", pattern, "$"), replacement, char)
+  }
+
+  # Replace the pattern if it is surrounded by "{}"
+  .replaceInVariableName <- function(pattern, replacement, char) {
+    sub(paste0("\\{", pattern, "\\}"), paste0("\\{", replacement, "\\}"), char)
+  }
+
+  # Replace occurences of "ren" and "ren.renr" by the correct column names
+  .replaceRenr <- function(char, replaceFunc) {
+    char <- replaceFunc("ren\\.renr", paste(initialCol, finalCol, sep = "."), char)
+    char <- replaceFunc("ren", initialCol, char)
+    char <- replaceFunc("renr", finalCol, char)
+    char <- replaceFunc("nonRen", setdiff(c("bs", "hs"), initialCol), char)
+  }
```

Comment:
I find implicit returns a bit harder to read.
```suggestion
  char
  }
```